### ISSUE TITLE
Spelling mistake in pb-cred.xhtml

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#6441]
|[<img alt="itamarst's avatar" src="https://avatars.githubusercontent.com/u/3266662?s=50" width="50" height="50">](https://github.com/itamarst)| @itamarst reported|
|-|-|
|Trac ID|trac#6441|
|Type|defect|
|Created|2013-04-17 13:21:18Z|

`doc/core/howto/pb-cred.xhtml` says "Futhermore" rather than "Furthermore".

Attachments:

* [pb-cread_xhtml.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/10a/10a5fad164f122ccc309f3cd60d6a402b2094eef/5c598054e70d6d325a9b8e2390ec8d511071032e.patch) (664 bytes) - added by chitrankdixit on 2013-04-21 05:45:53Z - spelling mistake "Futhermore" corrected to "Furthermore"
* [pb-cred.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/10a/10a5fad164f122ccc309f3cd60d6a402b2094eef/26ed55d96588995391868f1936840a0f50c8d5e4.patch) (664 bytes) - added by acalba on 2013-04-21 20:08:41Z - Adding patch with Spelling Error fixed

<details><summary>Searchable metadata</summary>

```
trac-id__6441 6441
type__defect defect
reporter__itamar itamar
priority__lowest lowest
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__easy easy
time__1366204878000000 1366204878000000
changetime__1366914593000000 1366914593000000
version__None None
owner__thijs thijs

```
</details>

**Comment by twisted-trac** [comment]
|<img alt="acalba's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|acalba set owner to acalba<br>acalba set status to `assigned`|
|-|-|

**Comment by twisted-trac** [comment]
|<img alt="chitrankdixit's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|chitrankdixit removed owner<br>chitrankdixit set status to `new`|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_4"></a>|@thijstriemstra set owner to @thijstriemstra<br>@thijstriemstra set status to `assigned`|
|-|-|

Thanks for your patch, I'll apply it. Make sure to create your patch from the root of your checkout next time, as described [TwistedDevelopment#SubmittingaPatch here].

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_5"></a>|@thijstriemstra set status to `closed`|
|-|-|

(In [38258]) Apply pb-cred.patch: Correct a typo in the perspective broker authentication howto.

Author: chitrankdixit
Reviewer: thijs
Fixes: [#6441](https://github.com/twisted/twisted/issues/6441)
